.github/CODEOWNERS

```diff
@@ -167,6 +167,7 @@ rush.json       @calebmshafer @scsewall
 /test-apps/imodel-from-geojson              @mgooding
 /test-apps/imodel-from-orbitgt              @RBBentley
 /test-apps/imodel-from-reality-model        @RBBentley
+/test-apps/imodel-transformer               @scsewall

```

Comment:
mind adding one more person to the list?  I've been meaning to go back and adding someone else to all the other test-apps.

Reply:
@calebmshafer I chose @mgooding as a backup CODEOWNER of `imodel-transformer`.